### PR TITLE
FIX: Word Matching

### PR DESCRIPTION
**Pull request by jromovel**
This update replaces the substring matching with regex word matching which was identified as a bug in this issue: https://github.com/afringer/Haku/issues/12

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const util = require("util");
 const Discord = require("discord.js");
-const config = require("./config-dev.js");
+const config = require("./config.js");
```
Comment (abrenner) [inline index.js]: Create a symlink on your system so you do not have to change the code.

config.js is a symlink that will point to config-dev.js

File: index.js [hunk 2]
```diff
@@ -38,9 +38,10 @@ client.on("message", (msg) => {
 		msg.channel.send(randomString(responses.KristenMentions));
```
Comment (abrenner) [inline index.js]: Don't we need a regex for this line?

File: index.js [hunk 3]
```diff
@@ -88,21 +89,17 @@ client.on("message", (msg) => {
...
 		msg.channel.send("I have a " + randomString(responses.gourdList));
 		return;
 	}
 
-	if (
-		(c.includes("friends") || c.includes("friend")) &&
-		(c.includes("message") || c.includes("messages")) &&
-		msgAuthorId === KristenUserId
-	) {
+	if (c.match(/\bfriends* messages*\b/) && msgAuthorId === KristenUserId) {
```
Comment (abrenner) [inline index.js]: Would the following phrase trigger the bot?

`Haku, show me messages from my friends!`

File: index.js (hunk 1 above)
Comment (jromovel) [reply index.js]: got it

File: index.js (hunk 2 above)
Comment (jromovel) [reply index.js]: yes, in order to be consistent

File: index.js (hunk 3 above)
Comment (jromovel) [reply index.js]: no it won't.
need to fix this

**Pull request merged**